Source/WebCore/dom/Document.cpp

```diff
@@ -8907,7 +9067,24 @@ void Document::mediaStreamCaptureStateChanged()
     });
 }
 
-#endif
+void Document::updateVideoCaptureAccordingMicrophoneInterruption(bool isMicrophoneInterrupted)
+{
+    RELEASE_LOG_INFO(WebRTC, "Document::updateVideoCaptureAccordingMicrophoneInterruption %d", isMicrophoneInterrupted);
+
+    RefPtr page = this->page();
+    if (!page)
+        return;
+
+    for (auto& document : allDocuments()) {
```

Comment:
shouldn't we take a strong reference here just in case? seems to have been a generic trend lately

Reply:
I'll add it.